packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -477,6 +478,25 @@ async function handler(
     }
   }
 
+  const availableSlots = await getAvailableSlots({
```

Comment:
getAvailableSlots and later on ensureAvailableUsers both call getUsersAvailability internally. 
Can you confirm it does not adversely affect the performance of this endpoint? 

You should also compare it with doing it by adding guests to users that are passed to ensureUsersAvailabilityo